src/ng2-smart-table.module.ts

```diff
@@ -41,3 +41,5 @@ import { TitleComponent } from './ng2-smart-table/components/title/title.compone
 })
 export class Ng2SmartTableModule {
 }
+export { Cell } from './ng2-smart-table/lib/data-set/cell';
```

Comment:
let's move the global exports to https://github.com/akveo/ng2-smart-table/blob/master/ng2-smart-table.ts